pr_agent/settings/configuration.toml

```diff
@@ -25,6 +25,9 @@ inline_code_comments = false
 ask_and_reflect=false
 automatic_review=true
 remove_previous_review_comment=false
+require_all_thresholds_for_incremental_review=false
```

Comment:
can you stack all the 'incremental' parameters ?
something like

# incremental parameters:
minimal_commits_for_incremental_review=0
minimal_minutes_for_incremental_review=0
...